src/QsCompiler/QirGeneration/ScopeManager.cs

```diff
@@ -158,18 +158,21 @@ public void RegisterVariable(string varName, IValue value)
             /// and makes sure the release function is invoked before unreferending the value.
```

Comment:
Now that release registration has been separated into its own function, this comment should be updated.

Reply:
Thanks! I missed that.